config/models.php

```diff
@@ -183,7 +183,7 @@
         |
         */
 
-        'snake_attributes' => true,
+        'snake_attributes' => false,
```

Comment:
Would you mind explaining why are we changing this line?
To give a little context on why `true` is the default value: It's [Laravel's default value](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php#L117). This means that most projects might have its value as true from the base Eloquent class.

Reply:
sorry i will fix that , i was testing the camelCase naming